examples/v0.6.0/httpscaledobject.yaml

```diff
@@ -8,7 +8,7 @@ spec:
     pathPrefixes:
     - /test
     scaleTargetRef:
-        deployment: xkcd
+        name: xkcd
```

Comment:
We shouldn't change this. The version 0.6.0 uses `deployment`

Reply:
good catch, you are right, I don't know why I included this change? 🤔 probably a mistake

Reply:
Fixed